helloworld-mutual-ssl-secured/configure-ssl.cli

```diff
@@ -4,7 +4,7 @@
 batch
 
 # Add the keystore and trust manager configuration in the elytron subsystem
-/subsystem=elytron/key-store=qsTrustStore:add(path=client.truststore,relative-to=jboss.server.config.dir,type=JKS,credential-reference={clear-text=secret})
+/subsystem=elytron/key-store=qsTrustStore:add(path=server.truststore,relative-to=jboss.server.config.dir,type=PKCS12,credential-reference={clear-text=secret})
```

Comment:
I think we have truststore configuration in two places now, in configure-certs.cli and here too with different names for the server's truststore.

Reply:
I don't think so. 
The server.truststore here is referring to the truststore created here. 
https://github.com/wildfly/quickstart/blob/a0d4b947f38bcb21382e2a37e3af4039a8bee2fb/helloworld-mutual-ssl-secured/configure-certs.cli#L10-L11
Initially the README instructions created client.truststore, which confusingly held the client's certificate and was actually the server's truststore. So, I changed it to server.truststore to be more clear.

Reply:
Why do we need both qsTrustStore and serverTS?

Reply:
serverTS is used in configure-client-cert.cli to create the server's truststore. 
However, this QS also includes instructions to generate these keystores and truststores on the terminal manually.
When they generate the truststore manually, the serverTS keystore would not exist and the script would fail, which is why I did not get rid of qsTrustStore.

Reply:
What if we update the README so that the manual instructions use serverTS as well? Then we wouldn't need both right? We should make sure the names used in the README match what we are using in the scripts.

Reply:
I am not sure I understand. 
The manual case uses keytool to generate the client keystore and the server truststore from it using
`$>keytool -importkeystore -srckeystore client.keystore -srcstorepass secret -destkeystore clientCert.p12 -srcstoretype PKCS12 -deststoretype PKCS12 -deststorepass secret`
So, there is no server configuration yet. (the .cli script is only run if they want to skip using keytool)

If there is a way to do conditional statements in a .cli script then we can so something like 

```
if (outcome != success) of ./subsystem=elytron/keystore=serverTS:read-resource
Add quickstartTS
else
echo INFO: server truststore is already in configuration, not added.
end-if
```
is this something we can do? with this we can rename serverTS into quickstartTS in the configure-client-cert.cli script.

Reply:
Users following the standard distribution path still need to run the cli script to create the certs manually, and such instructions should replace the old using the multiple OS commands.

Reply:
@PrarthonaPaul you can use conditionals on cli scripts, examples found on other quickstarts:
```
if (result == undefined) of /socket-binding-group=standard-sockets/socket-binding=http:read-attribute(name=client-mappings)
if (outcome == success) of /subsystem=messaging-activemq:read-resource

```

Reply:
I have updated the readme and CLI scripts. 
Since the instructions only use the cli script now, there is no need to do the conditional statements in the script anymore. 
Thanks!